### PR TITLE
jv_homework_jdbc

### PR DESCRIPTION
**Pull request by etezaadi**
Connected to db;
Created the Manufacturer model;
Created and implemented dao layer for the Manufacturer model;
Used dao annotation and injector;
Created init_db.sql.

### CODE REVIEW

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            createManufacturerStatement.setObject(3, manufacturer.getId());
+            createManufacturerStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer from DB "
+                    + manufacturer, e);
+        }
+        return manufacturer;
+    }
+
+    private static Manufacturer parseManufacturer(ResultSet resultSet) throws SQLException {
```
Comment (Nick97-git) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:111]:
```suggestion
    private Manufacturer parseManufacturer(ResultSet resultSet) throws SQLException {
```

File: src/main/java/mate/jdbc/util/ConnectionUtil.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "12345";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String URL_CONNECTIVITY = "jdbc:mysql://localhost:3306/taxi-service_db";
```
Comment (Nick97-git) [inline src/main/java/mate/jdbc/util/ConnectionUtil.java:12]:
```suggestion
    private static final String JDBC_URL = "jdbc:mysql://localhost:3306/taxi-service_db";
```

File: src/main/resources/init_db.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+CREATE DATABASE `taxi-service_db` /*!40100 DEFAULT CHARACTER SET utf8mb3 */ /*!80016 DEFAULT ENCRYPTION='N' */;
```
Comment (Nick97-git) [inline src/main/resources/init_db.sql:1]:
```suggestion
CREATE DATABASE `taxi_service_db` 
```

File: src/main/resources/init_db.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+CREATE TABLE `manufacturers` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) DEFAULT NULL,
+  `country` varchar(45) DEFAULT NULL,
+  `is_deleted` tinyint NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=17 DEFAULT CHARSET=utf8mb3;
+
+
```
Comment (Nick97-git) [inline src/main/resources/init_db.sql:11]: remove empty lines

File: src/main/resources/init_db.sql [hunk 5]
```diff
@@ -0,0 +1,11 @@
+CREATE DATABASE `taxi-service_db` /*!40100 DEFAULT CHARACTER SET utf8mb3 */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
+CREATE TABLE `manufacturers` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) DEFAULT NULL,
+  `country` varchar(45) DEFAULT NULL,
+  `is_deleted` tinyint NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=17 DEFAULT CHARSET=utf8mb3;
```
Comment (Nick97-git) [inline src/main/resources/init_db.sql:9]:
```suggestion
) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb3;
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Comment by etezaadi** [comment]
@Nick97-git thanks for support, already fixed

**Pull request closed without merging**